cache/cache.go

```diff
@@ -66,44 +97,73 @@ func newProject(name, version string) (*Project, error) {
 	}
 
 	nwo := strings.SplitN(name, "/", 2)
-	if version == "" {
+	if versionString == "" {
 		ev := os.Getenv(fmt.Sprintf("%s_VERSION", strings.ToUpper(nwo[1])))
 		if ev == "" {
 			return nil, errors.Errorf("unknown project version for %s", name)
 		}
-		version = ev
+		versionString = ev
+	}
+
+	if !strings.HasPrefix(versionString, "v") {
```

Comment:
we need both versions, with and without the `v`. GitHub uses the version with `v` in their path, but then, tarballs might not have it. 😞